tests/test_jobs.py

```diff
@@ -50,6 +50,7 @@
     'dispatcher_token': 'tokened',
     'hq_url': 'http://example.com/hq',
     'hq_token': 'tokened',
+    'worker_host': 'albus.frigg.io'
```

Comment:
It is used on line [71 in this file](https://github.com/frigg/frigg-worker/pull/80/files#diff-ee26cc8025d70ef09f379603aa48a64fR71)